enterprise/kernel/src/main/java/org/neo4j/kernel/enterprise/builtinprocs/EnterpriseBuiltInDbmsProcedures.java

```diff
@@ -83,20 +80,20 @@
     public void setTXMetaData( @Name( value = "data" ) Map<String,Object> data )
     {
         securityContext.assertCredentialsNotExpired();
-        int totalCharSize = data.entrySet().stream()
-                .mapToInt( e -> e.getKey().length() + e.getValue().toString().length() )
-                .sum();
-
-        if ( totalCharSize >= HARD_CHAR_LIMIT )
```

Comment:
I removed it since most of the usages of metadata that I saw was by actually working with metadata map in TKI directly bypassing procedure to set values and that means that this check does not really solve it purpose completely, and instead add weirdness of failing depending on your toString() result size. But i'm fully trust you here so will restore this check